packages/v4/src/content/design-guidelines/usage-and-behavior/toolbar/toolbar.md

```diff
@@ -9,147 +9,126 @@ Toolbars can be included to allow a user to manage and manipulate a data set. Da
+The toolbar is a container for holding a variety of elements (toolbar items and groups) that are used to manipulate a set of data. All items should be centered vertically within the toolbar. There is a 24px of padding before and after the toolbar content.
 
-![toolbar](./img/basic-toolbar-nocallout.png)
+<img src="./img/toolbar-spacing.png"  alt="basic toolbar with 16px spacing"  width="1210"/>
 
-All items should be centered vertically within the height.
+Toolbar items are individual components that can be placed inside of a toolbar. Often, it makes sense to group sets of like items to create desired associations and to enable items to respond together to changes in viewport width. The default spacing between items and groups is 16px, but this can be modified, if required. The following are some toolbar items and groups that come with special spacing considerations.
 
-### Toolbar items and groups
-Toolbar items are individual components that can be placed inside of a toolbar. Buttons, select lists, and pagination controls are examples of items. Often, it makes sense to group sets of like items to create desired associations and to enable items to respond together to changes in viewport width.
+#### Bulk selector
+<img src="./img/bulk-selector.png" alt="bulk selector item" width="218" />
 
-Below are examples of: a group of icons, a group of buttons, a group of icons and buttons.
+ A Bulk selector can be added when you want to give users the ability to select multiple items within a page. Besides allowing the user to select and deselect all items on a page, other application specific options can be added in the dropdown. Bulk selection allows the user to select or deselect all items on a page and provide an indeterminate state that notifies the user when only some items are selected. Bulk selectors should always be the left-most element in a toolbar and has a 24px spacer to its right.
 
-![toolbar items and groups](./img/items-and-groups.png)
+ #### Filter group
+ <img src="./img/filter-group.png" alt="filter group" width="355" />
 
-Items within a group have 16px spacing between them. Items that are made up of multiple parts (for example the filter dropdown and input group) are treated as one item in regards to the spacing.
+  A filter group combines multiple select filters into a single group. See [Filters](/design-guidelines/usage-and-behavior/filters) for more information about using filters.
 
+#### Search filter
+<img src="./img/search-filter.png" alt="search filter" width="355" />
 
-### Multiple Groups
+A search filter is intended to be used with a filter group as shown below when you want to include a single search field together with one or more select filters. It include an 8px spacer to the right of the item to enforce association with other filters. See the [Filters](design-guidelines/usage-and-behavior/filters) documentation for more information about using filters.
```

Comment:
Found that include should be changed to includes - "It includes an 8px spacer..."